### PR TITLE
feat: new line before expect

### PR DESCRIPTION
**Pull request by DianaSuvorova**
issue #50

### CODE REVIEW

File: docs/rules/new-line-before-expect.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Rule details
+
+This rule triggers a **warning** (is set to **1** by default) whenever there is no new line before expect.
+
+The following patterns are considered warnings:
+
+```js
+describe("", function() {
+  it("", function() {
+    expect(1).toBe(1);
```
Comment (burabure) [inline docs/rules/new-line-before-expect.md]: I think if the expect is the first line on the `it` block it shouldn't be a warning

File: lib/helpers/isPaddingBetweenTokens.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+* Checks if there is padding between two tokens
+* @param {Token} first The first token
+* @param {Token} second The second token
+* @returns {boolean} True if there is at least a line between the tokens
+*/
+function isPaddingBetweenTokens (first, second) {
```
Comment (burabure) [inline lib/helpers/isPaddingBetweenTokens.js]: maybe `hasPaddingBetweenTokens` is better?

File: docs/rules/new-line-before-expect.md (hunk 1 above)
Comment (DianaSuvorova) [inline docs/rules/new-line-before-expect.md]: updated.

**Comment by burabure** [comment]
Thanks for your PR! Merging this now :tada:

**Pull request merged**